apps/documentation/specs/index.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
We might want to remove unit tests, since most of our pages should be straight-forward.